### PR TITLE
Fixes #81: Added link of the location 

### LINKED ISSUE #81: Map Marker
map marker should have a link to the following location

### PR DESCRIPTION
**Pull request by samyakbvs**
Added the location to the Map marker 
Issue #81

### CODE REVIEW

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/samyakbvs-2012.fossasia.org/
Test link

**Comment by vaibhavsingh97** [comment]
Link not working. Please check

**Comment by samyakbvs** [comment]
Ya lemme send it again its not working bcs I changed the repo  name

**Comment by samyakbvs** [comment]
@vaibhavsingh97 Here you go... 
https://samyakbvs.github.io/samyak-ja-in-bvs-2012.fossasia.org/

File: _includes/footer.html [hunk 1]
```diff
@@ -22,7 +22,8 @@
                             <ul class="contact-methods">
                                 <li><i class="icon pe-7s-mail"></i><span><a id="mailAdd" href="mailto:fossasia@googlegroups.com" target="_self" style="color:#777777">fossasia@googlegroups.com</a></span></li>
                                 <li><i class="icon pe-7s-phone"></i><span>+65 83435672</span></li>
-                                <li><i class="icon pe-7s-map-marker"></i><span>FOSSASIA</span></li>
+                                        <li><a href="https://www.google.co.in/maps/search/6+Eu+Tong+Sen+Street+%2305-07+The+Central,+Singapore+059817/@1.2889009,103.8443213,17z/data=!3m1!4b1?hl=en">
```
Comment (vaibhavsingh97) [inline _includes/footer.html]: Hi @samyakbvs No one going to click on the map_marker icon. You can add it to the address given

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

File: _includes/footer.html (hunk 1 above)
Comment (samyakbvs) [reply _includes/footer.html]: okay no problem

**Comment by samyakbvs** [comment]
@vaibhavsingh97 
Done

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
Link is not visible. Please check and fix

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/s-a-my-ak-ja-in-bvs-2012.fossasia.org/

**Comment by samyakbvs** [comment]
@vaibhavsingh97 changes made

**Comment by akshatnitd** [comment]
@samyakbvs Please squash your commits and it will be ready to merge!

**Review by akshatnitd — COMMENTED** [review]
@samyakbvs Please squash your commits and it will be ready to merge!

**Comment by samyakbvs** [comment]
@akshatnitd I won't be able to squash as I had committed these changes a few days ago and now other commits have come on top of it so I won't be able to squash so pls merge like this

**Comment by harshit98** [comment]
@samyakbvs 

It's okay if you're not able to squash commits. Change your PR name to `Fixes #(issue-num): details of fix`. Since you've done a lot of contribution, we would like you to follow best practices as well. Thanks!

**Review by harshit98 — APPROVED** [review]

**Comment by mariobehling** [comment]
More color and animation is starting to confuse the user. The FOSSASIA sites should be lean and simple and there should not be more and more elements introduced, but rather align with the existing design following fossasia.org. I will have to revert this.

**Pull request merged**